src/translations/en.json

```diff
@@ -726,7 +726,19 @@
           "picker": {
             "header": "Customization",
             "introduction": "Tweak per-entity attributes. Added/edited customizations will take effect immediately. Removed customizations will take effect when the entity is updated."
-          }
+          },
+          "warning": {
+            "include_sentence": "It seems that your configuration.yaml doesn't properly",
+            "include_link": "include customize.yaml",
```

Comment:
We should bring include to the other key and include the link as a variable in that string.

Reply:
I am not quite sure what you mean by that. Do you want to pass the entire `<a>` into one translation string?

Reply:
Yes

Reply:
Like this?
```json
"include_sentence": "It seems that your configuration.yaml doesn't properly {link}"
```

Reply:
I would do:
`"include_sentence": "It seems that your configuration.yaml doesn't properly include {customize_link}"`

Reply:
Oddly enough, I cannot pass the entire `<a>` into the string:
```js
[[localize('ui.panel.config.customize.warning.include_sentence',
          'customize_link',
          "<a
          href="https://www.home-assistant.io/docs/configuration/customizing-devices/#customization-using-the-ui"
          target="_blank"
          >configuration.yaml</a>")]]
```
Just prints it out like that as the innerHTML of the `div` element. 
Does the `localize` funciton filter html elements?

Reply:
yeah probably... OK, never mind what I said.

Reply:
So how should I address this? Just leave it like that or is there any other option?